src/settings/project/mod.rs

```diff
@@ -96,14 +62,23 @@ impl Project {
     }
 
     pub fn new() -> Result<Self, failure::Error> {
-        get_project_config()
+        let mut config_path = PathBuf::new();
+        config_path.push("./wrangler.toml");
+
+        get_project_config(config_path)
+    }
+
+    pub fn kv_namespaces(&self) -> Vec<KvNamespace> {
+        match &self.kv_namespaces {
+            Some(kv) => kv.clone(),
+            None => Vec::new(),
+        }
     }
 }
 
-pub fn get_project_config() -> Result<Project, failure::Error> {
+fn get_project_config(config_path: PathBuf) -> Result<Project, failure::Error> {
```

Comment:
usually, functions take a `&Path` instead of a `PathBuf`, and it looks like `config_path` was previously a `&Path`; this should probably change to that